tests/common/dualtor/dual_tor_io.py

```diff
@@ -198,12 +237,13 @@ def generate_from_server_to_t1(self):
         @summary: Generate (not send) the packets to be sent from server to T1
         """
         eth_src = self.ptfadapter.dataplane.get_mac(0, 0)
-        if self.server_port:
-            self.from_server_src_port = self.server_port
+        if self.tor_vlan_port:
+            from_server_src_port = self.tor_vlan_port
         else:
-            self.from_server_src_port = random.choice(self.vlan_ports)
-        self.from_server_src_addr  = random.choice(self.vlan_host_map[self.from_server_src_port].keys())
-        self.from_server_dst_addr  = self.random_host_ip()
+            from_server_src_port = random.choice(self.vlan_ports.values())
```

Comment:
I like what you did in `send_t1_to_server` where every packet was sent to a different server, instead of randomly picking one server and using it for all packets. Could we do the same thing here, if `tor_vlan_port` is not specified? That way we can send traffic across multiple mux cables instead of just the one.

Reply:
Good idea! Decided to do this in a later PR, and merge this change as these are critical for data plane testing.